x-pack/plugins/index_management/public/application/sections/home/index_list/details_page/details_page_overview/aliases_details.tsx

```diff
@@ -0,0 +1,113 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React, { FunctionComponent, useState } from 'react';
+import { css } from '@emotion/react';
+import { euiThemeVars } from '@kbn/ui-theme';
+import { i18n } from '@kbn/i18n';
+import {
+  EuiBadge,
+  EuiBadgeGroup,
+  EuiButton,
+  EuiFlexGroup,
+  EuiFlexItem,
+  EuiFlyout,
+  EuiFlyoutBody,
+  EuiFlyoutHeader,
+  EuiListGroup,
+  EuiListGroupItem,
+  EuiText,
+  EuiTextColor,
+  EuiTitle,
+} from '@elastic/eui';
+
+import { Index } from '../../../../../../../common';
+import { OverviewCard } from './overview_card';
+
+export const AliasesDetails: FunctionComponent<{ aliases: Index['aliases'] }> = ({ aliases }) => {
+  const [isShowingAliases, setIsShowingAliases] = useState<boolean>(false);
+  if (!Array.isArray(aliases)) {
+    return null;
+  }
+  const aliasesBadges = aliases.slice(0, 3).map((alias) => <EuiBadge>{alias}</EuiBadge>);
```

Comment:
Should we truncate the text for each alias? if some of them are long it might look a bit off in the ui

![Screenshot 2023-10-27 at 13 23 54](https://github.com/elastic/kibana/assets/1191206/80ce6f0b-9eb6-4dab-a2fb-e09abf998202)

Reply:
Great point, I think I could set a sensible max width value, something like 250px